src/globus_sdk/services/flows/client.py

```diff
@@ -327,8 +342,15 @@ def list_flows(
             query_params = {}
         if filter_role is not None:
             query_params["filter_role"] = filter_role
```

Comment:
We discussed adding a deprecation warning here (we have tooling for this, use the `warn_deprecated` helper), which I think we're on solid ground to include. It's not clear when this is being removed from the API, but adding the warning gives us better grounds to remove the parameter in v4 and have a simpler SDK interface.